fjord/feedback/tests/test_views.py

```diff
@@ -115,68 +117,60 @@ def test_valid_sad(self):
         eq_(u'14.0.1', feedback.version)
 
     @with_waffle('thankyou', True)
-    def test_use_thank_you_sad(self):
-        """Thanks_sad template is used when appropriate"""
+    def test_thankyou_flag_active(self):
+        """Verify response and suggestions when thankyou flag is active"""
         url = reverse('feedback', args=(u'firefox',), locale='en-US')
         r = self.client.post(url, {
             'happy': 0,
-            'description': u"I want to know why Firefox doesn't make me sandwiches!",
+            'description': u'Why Firefox not make me sandwiches!',
         }, follow=True)
 
         feedback = models.Response.objects.latest(field_name='id')
-        eq_(u"I want to know why Firefox doesn't make me sandwiches!", feedback.description)
-        eq_(u'', feedback.url)
-        eq_(False, feedback.happy)
-        eq_(u'en-US', feedback.locale)
         eq_(r.status_code, 200)
-        self.assertTemplateUsed(r, 'feedback/thanks_sad.html')
+        eq_(r.context['response'].id, feedback.id)
```

Comment:
This is checking the context that was passed to render the template? I didn't know you could do that. Awesome!